tests/integration-tests/tests/update/test_update.py

```diff
@@ -837,6 +841,36 @@ def _test_update_queue_strategy_with_running_job(
     assert_compute_node_states(scheduler_commands, queue1_nodes, "idle")
 
 
+@pytest.fixture
+@pytest.mark.usefixtures("parameterized_cfn_stacks_factory")
+def external_shared_storage_stack(request, test_datadir):
+    def _create_stack(region, vpc, PublicAz3, PublicAz2, import_path, export_path, Public):
+        template_path = os.path.join(str(test_datadir), "storage-stack.json")
```

Comment:
I expect the PR to have the `storage-stack.json` file too, but it is currently missing.
Furthermore, why using JSON instead of YAML? YAML would improve a lot the readability of the template.
In any case, may you please commit it and include in the PR?

Reply:
Sorry, I thought that was already pushed. I just pushed it. It is json because that was the format the separate template files came in from cloud formation.

Reply:
Ok, also I used the template designer from the cfn console for the conversion. I'll push that in my next commit.